### ISSUE TITLE
Improve instructions for filling in recommendation acceptance form

### ISSUE THREAD

**Issue by KirstieJane** [KirstieJane/STEMMRoleModels#102]
I'm going to use this issue to list the manual steps that I have to take in order to wrangle the submitted responses [from this form](https://docs.google.com/forms/d/e/1FAIpQLSfehG4p2Ub5NFBPS88Gsz03Zfeno8HCMSaP7ODw-vh5BhdrrQ/viewform) into the right format for the data sheet.

If anyone has any suggestions on how to make the instructions clearer please add them here!
- [ ] Title given as "PhD" when field is expecting "Dr" or "Prof".
- [ ] Title given with period after it, eg: Mr. rather than Mr (which isn't wrong but leads to mixed styles)
- This is used before the role model's name in the main page of the database.
- Here's the current text: _What is your title? (Prof, Dr, Ms, etc)_
  <br>
  <br>
- [ ] State given in both abbreviation and long form ("NY - New York")
- This is used in the location below the role model's name in the main page of the database.
- Here's the current text: _In which state do you work?_
  <br>
  <br>
- [ ] Country given in both abbreviation and long form by different role models("UK" and "United Kingdom")
- This is used in the location below the role model's name in the main page of the database and used to count the number of countries we have represented in the database.
- Here's the current text: _In which country do you work?_
  <br>
  <br>
- [ ] Picture provided as link to dropbox image rather than to the raw image itself.
- [ ] Picture provided as link to google image rather than to raw image itself.
- This prevents the image from showing up if the code can't recognize it as a picture.
- Here's the current text: _Please include a link to a photograph of you_
- There's also some information [in the FAQs](http://stemmrolemodels.com/site/faq.html#how-do-i-add-a-profile-photo) about this.
- **ACTION:** @KirstieJane added link to FAQ text in form....we'll see if that helps!

**Comment by KirstieJane** [comment]
Hey @neuroAmyo - I've started this list based on how you (the very first user!) filled it in. Take a look and see if there are any wording etc changes that you think would have helped!

**Comment by Ryuno-Ki** [comment]
I've not a current setup around, but couldn't the mixed format be dealt with by offering viable titles as dropdown?

**Comment by KirstieJane** [comment]
Yep - that's definitely one way of doing it :smile:

**Comment by neuroAmyo** [comment]
I agree with @Ryuno-Ki that a drop-down list for titles seems best. I, for instance, did Ph.D. rather than Dr because I am honestly always confused by whether Dr is for medical doctors or for anyone with a Ph.D.! We can always have an "other" option with a write-in so that we're inclusive. 

The state thing was because Google auto-completed part of the form for me :). A list for this is a little harder, right? Because for a global website, there are tons of options. Surely someone who knows more about making web-forms could help with this. I would be surprised if there aren't already available options to help with location-based lists etc. 

The link issue is an interesting one because we don't want a certain level of tech/web proficiency to add a barrier. I am not the most web-savy person and definitely would never realize that a dropbox link to a picture is different from a link to a raw image on the web.

**Comment by KirstieJane** [comment]
@neuroAmyo - I think at the moment it's totally fine that there's a manual edit (by me). The scaling up (eg: by changing how we host the database - ie not through google docs, or by uploading a picture) is probably the best way to solve these problems.

I mean for this issue to be 1) a requirement gathering collection of what we need to change/make easier when/if we have time to develop the next gen of our database, or 2) ways to collect "quick fixes" for clarification of questions.
